web-app/.gitignore

```diff
@@ -41,3 +41,4 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+../.DS_Store
```

Comment:
Don't think this is needed.

Reply:
It's a mac thing I'm pretty sure

Reply:
Did you add this yourself or was it auto generated?

Reply:
I added this myself